app/assets/javascripts/accordion-with-descriptions.js

```diff
@@ -0,0 +1,327 @@
+// Most of this is taken from the service manual and at some point needs to be
```

Comment:
Is this just lifted and shifted, or did you have to change anything?

Reply:
My initial thought is that this file was going to be needed to implement the sidebar functionality. I'm still not sure if the frontend developer will need it, but this makes it easier to have it on the prototype if needed.